packages/cozy-procedures/package.json

```diff
@@ -21,6 +21,7 @@
   "dependencies": {
     "@babel/runtime": "^7.5.4",
     "babel-preset-cozy-app": "^1.6.0",
+    "date-fns": "1.30.1",
```

Comment:
Je mettrais "^1.30.1" ici vu qu'on est dans une librairie. https://renovatebot.com/docs/dependency-pinning/#ranges-for-libraries